adr/0023-mid-cycle-reporting.md

```diff
@@ -0,0 +1,28 @@
+# 23. Mid-cycle Reporting
```

Comment:
We know this naming is misleading, so perhaps we should start referring to it as "Per-provider reporting" at the high-level. Essentially, "mid-cycle report" is what it was, and now we are creating something better.

Reply:
Resolved in d697f733